content/deploy/community-cloud/status-and-limitations.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Status and limitations
+slug: /deploy/streamlit-community-cloud/status
+---
+
+# Status and limitations of Community Cloud
+
+## Community Cloud Status
+
+You can view the current status of Community Cloud at [streamlitstatus.com](https://www.streamlitstatus.com/).
+
+## GitHub OAuth scope
+
+To deploy your app, Streamlit requires access to your app's source code in GitHub and the ability to manage the public keys associated with your repositories. The default GitHub OAuth scopes are sufficient to work with apps in public GitHub repositories. However, to access your private repositories, we create a read-only [GitHub Deploy Key](https://docs.github.com/en/free-pro-team@latest/developers/overview/managing-deploy-keys#deploy-keys) and then access your repo using a public SSH key. When we create this key, GitHub notifies repo admins of the creation as a security measure.
```

Comment:
I think the public part of the key is stored in GitHub and we store and use the private part.